hm_pyhelper/tests/data/sentry/sample_exception_event.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
Ignore the json files. They are just sample dumps sentry events to serve as test data.